quota/etcd/quotapb/quotapb.proto

```diff
@@ -133,6 +133,10 @@ message UpdateConfigRequest {
   // Fields modified by the update request.
   // For example: "state" or "max_tokens".
   google.protobuf.FieldMask update_mask = 3;
+
+  // If true the updated quota is reset, regardless of the updates's contents.
+  // A reset quota is replenished to its maximum number of tokens.
+  bool reset_quota = 4;
```

Comment:
This doesn't feel like it belongs here -- it doesn't change the *configuration* of a quota, it changes the current state.  Maybe add a top-level `ResetQuotaRequest` instead?